### PR TITLE
Restructure Clear Linux content

### PR DESCRIPTION
**Pull request by iphutch**
This restructure organizes the Clear Linux content into five sections: Get Started, Concepts, Guides, Tutorials, and Reference. File names and doc-trees were changed where appropriate along with labels to ensure they remained standard. Where labels were changes, cross-references were also checked and changed if appropriate. Content was left as untouched as possible, however some changes were necessary to allow readers to flow from concepts into guides.

Signed off by: Ianeta Hutchinson <ianeta.hutchinson@intel.com>

### CODE REVIEW

**Comment by iphutch** [comment]
@tpleavitt @rcaballeromx Thoughts on the guide titles? I only have Network in there and would like to better categorize the other tasks.

File: source/clear-linux/get-started/get-started.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.. _getting-started:
+.. _get-started:
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/get-started.rst]: Changing this label means that all \:ref\:\`getting-started\` instances must also be changed.

File: source/clear-linux/guides/network/dpdk.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _dpdk:
 
-DPDK
-####
+DPDK: Send packages between platforms
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/dpdk.rst]: These titles are too long which will cause problems with the navigation.

File: source/clear-linux/guides/network/network-bonding.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 .. _network-bonding:
 
-Network Bonding
-###############
+Network bonding: Configure systemd to use the bonding driver
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/network-bonding.rst]: ditto.

File: source/clear-linux/guides/network/network-boot.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .. _network-boot:
 
-Network Booting
-###############
+Network booting: Network boot with iPXE using NAT
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/network-boot.rst]: Ditto

File: source/clear-linux/guides/network/ovs-dpdk.rst [hunk 5]
```diff
@@ -1,7 +1,7 @@
 .. _ovs-dpdk:
 
-OpenvSwitch and DPDK
-####################
+DPDK: Enable DPDK support on OpenvSwitch
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/ovs-dpdk.rst]: I would get rid of the first DPDK since it is unneeded.

**Pull request closed without merging**